inc/zoo/swar/SWAR.h

```diff
@@ -184,6 +243,13 @@ constexpr auto isolateLSB(T v) {
 template<int NBits, typename T = uint64_t>
 constexpr auto lowestNBitsMask() {
   return (T(1)<<NBits)-1;
+  //return ~((~T(0))<<NBits);
+}
+
+template<int NBits, uint64_t T>
+constexpr auto lowestNBitsMask() {
```

Comment:
I think we should rename to `least` and `most` consistently, instead of `low`/`high`